tests/ssl-opt.sh

```diff
@@ -11893,14 +11892,13 @@ run_test    "TLS 1.3: Check signature algorithm order, m->O" \
                                  -msg -tls1_3 -num_tickets 0 -no_resume_ephemeral -no_cache
                                  -Verify 10 -sigalgs rsa_pkcs1_sha512:rsa_pss_rsae_sha512:rsa_pss_rsae_sha384:ecdsa_secp256r1_sha256" \
             "$P_CLI debug_level=4 crt_file=data_files/server2-sha256.crt key_file=data_files/server2.key \
-                    min_version=tls12 max_version=tls13 sig_algs=rsa_pkcs1_sha512,rsa_pss_rsae_sha512,rsa_pss_rsae_sha384,ecdsa_secp256r1_sha256" \
+                    force_version=tls13 sig_algs=rsa_pkcs1_sha512,rsa_pss_rsae_sha512,rsa_pss_rsae_sha384,ecdsa_secp256r1_sha256" \
```

Comment:
TLS 1.3 is forced on the server side in this test thus I'd expect that we do not need to force it on the client side as well. Am I missing something ? Ditto for the other instances.